cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -60,11 +60,11 @@
     <li class="list_item
                {{ _classes( nav_depth, '-item' ) }}"
         {{ 'data-js-hook=flyout-menu' if nav_children | count > 0 else '' }}>
-        {# TODO: Remove nav_depth < 2 check on has-children class when 3rd level transitions are in. #}
+        {# TODO: disable link (or overview link) of page user is currently on. #}
```

Comment:
Yeah, this is working on desktop, but not on mobile. I can clarify the `TODO`.
On mobile the problem is all third-level menus have a trigger link on the second-level. Those links happen to also be a link to the overview page. The logic I had in there made it so if you went to About us > Careers > Careers Overview. About us > Careers was disabled, not Careers Overview as it should be.